nixos/modules/services/web-servers/hydron.nix

```diff
@@ -77,18 +80,17 @@ in with lib; {
   };
 
   config = mkIf cfg.enable {
-    security.sudo.enable = cfg.enable == true;
-    services.postgresql.enable = cfg.enable == true;
-
-    services.hydron.passwordFile = mkDefault (toString (pkgs.writeTextFile {
-      name = "hydron-password-file";
-      text = cfg.password;
-    }));
-
-    services.hydron.postgresArgsFile = mkDefault (toString (pkgs.writeTextFile {
-      name = "hydron-postgres-args";
-      text = cfg.postgresArgs;
-    }));
+    security.sudo.enable = cfg.enable;
+    services.postgresql.enable = cfg.enable;
+    services.hydron.passwordFile = mkDefault (pkgs.writeText "hydron-password-file" cfg.password);
+    services.hydron.password = mkDefault ("hydron");
```

Comment:
No need to set the default here. The mkDefault thing is only needed when the default depends on another option value.